testsuite/features/secondary/allcli_software_channels.feature

```diff
@@ -136,7 +136,7 @@ Feature: Chanel subscription via SSM
     And channel "Test Child Channel" should be enabled on "sle_client"
 
 @centos_minion
-  Scenario: System default channel can't be determined
+  Scenario: System default channel can't be determined on the CentOS minion
```

Comment:
Not sure about it, but here again the mahine is called by centos_ssh_minion but the scenario says Centos minion

Reply:
Yes, we did not change all scenario names when we decided to move CentOS to SSH.

And we might revert that decision some day. The reason for moving to ssh was that it was unsufficiently tested at the time, but now it is, so it could be a good idea to shake the tree and revert to normal minions. What do you think?

Same applies to Ubuntu, everywhere.

Reply:
Ideally, I would use Salt minions as a default. SSH minions use to have fewer problems, as in the end, the server does more guided stuff in the client by connecting through ssh. So, yeah, maybe better have again salt minions.
Anyhow, I would reflect the kind of protocol we use in the title.

Reply:
Absolutely. I'll start an email thread as announced so everyone is fully aware of that change, but I think that's what we should do.

Reply:
Done.